lambda_function.py

```diff
@@ -18,16 +18,27 @@ def flat_out_filter(cfn_filter):
         flat_filter.append(condition)
     return flat_filter
 
+def flat_out_dict(d, flat=dict(), prefix=""):
+    """ Flat out dict so all attributes can be accessed with GetAtt """
+    for k,v in d.items():
+        if prefix == "":
+            key = str(k)
+        else:
+            key = prefix + "." + str(k)
+        if isinstance(v, dict):
+            flat_out_dict(v, flat, key)
+        else:
+            flat[key]=str(v)
+    return(flat)
 
 def handler(event, context):
     """Main Lambda Function"""
     response_data = {}
 
     log.debug("Received event: %s", json.dumps(event))
-    filters = flat_out_filter(event["ResourceProperties"]["Filter"])
+    try: filters = flat_out_filter(event["ResourceProperties"]["Filter"])
+    except KeyError: filters = []
```

Comment:
Not even sure that we need the `try` for setting default, fallback value :thinking: I have to double check that.